snapcraft/internal/lifecycle/_runner.py

```diff
@@ -260,29 +239,28 @@ def _restage(self, part, hint=''):
 
     def _run_prime(self, part):
         self._run_step(
-            step=steps.PRIME, part=part, progress='Priming',
-            prerequisite_step=steps.PRIME)
+            step=steps.PRIME, part=part, progress='Priming')
 
     def _reprime(self, part, hint=''):
         self._rerun_step(
-            step=steps.PRIME, part=part, progress='Re-priming', hint=hint,
-            prerequisite_step=steps.PRIME)
+            step=steps.PRIME, part=part, progress='Re-priming', hint=hint)
 
-    def _run_step(self, *, step: steps.Step, part, progress, hint='',
-                  prerequisite_step: steps.Step=steps.STAGE):
+    def _run_step(self, *, step: steps.Step, part, progress, hint=''):
         common.reset_env()
-        prereqs = self.parts_config.get_prereqs(part.name)
+        prereq_parts = self.parts_config.get_dependencies(part.name)
```

Comment:
so we had a nice conversation about `get_dependencies` being renamed, but the local var is still called `prereq_parts`?

Reply:
Yeah good catch, fixed.